Cargo.toml

```diff
@@ -36,8 +36,11 @@ futures = "0.3"
 tokio = "1"
 anyhow = "1"
 antidote = "1.0.0"
-http = "0.2"
-hyper = { version = "0.14", default-features = false }
+http = "1.0"
+hyper = { version = "1.0", default-features = false }
+hyper-util = { git = "https://github.com/hyperium/hyper-util" }
```

Comment:
This adds a dependency on a not-released crate. Personally, I feel like the update to hyper 1.0 should wait for a release of hyper-util

Reply:
hyper-util is now published, moved to official release

Reply:
awesome!